compiler/modules/nback/scaffold.bal

```diff
@@ -180,7 +180,7 @@ class Scaffold {
         self.reprs = reprs;
         self.returnType = defn.signature.returnType;
         self.retRepr = semTypeRetRepr(self.returnType);
-        self.nParams = defn.signature.paramTypes.length();
+        self.nParams = defn.signature.restParamType == () ? defn.signature.paramTypes.length() : defn.signature.paramTypes.length() + 1;
```

Comment:
Factor this into a function on FunctionSignature

Reply:
fixed (no longer required)